tlv493d/tlv493d.go

```diff
@@ -422,6 +422,18 @@ func (d *Dev) StopContinousRead() {
 	d.continuousReadWG.Wait()
 }
 
+// CalibrateTemperature sets the temperature offset compensation based upon a reading from the sensor compared to the actual temperature
```

Comment:
Since it should be set in Opts.TemperatureOffsetCompensation, I think it could be a stand alone function. This way people could calculate the value, then save it locally so it is specified in Opts, instead of changing the value once the device is initialized.

What do you think?

Reply:
Yes, I agree, however:
* the new value of Opts.TemperatureOffsetCompensation is returned as output of the method - but that's probably not explicit in the doc
* in order to extract the raw reading from the measured temperature, I need to use the Opts.TemperatureOffsetCompensation used for computing it, that's why I set this as a member function.

Basically, the function needs access to the raw value returned by the sensor and the matching temperature. So, what I can propose:
1. Make it more explicit in the documentation that the output of the function must be saved for reuse and put in Opts.TemperatureOffsetCompensation next time.
2. Make the function standalone and ask for Opts.TemperatureOffsetCompensation as input

I agree on doing 2, it makes sense. I'll do a PR shortly with it.

Thanks for the feedback!

Reply:
Great idea.